ddtrace/tracer.py

```diff
@@ -246,6 +248,25 @@ def current_trace_context(self, *args, **kwargs):
             return active.context
         return None
 
+    def get_log_correlation_context(self):
+        # type: () -> Optional[DDLogRecord]
```

Comment:
should this be `Optional`? probably better if we always return a `DDLogRecord`, just whether we have active values or not.

e.g. if we just did:

```python
span = None
if self.enabled:
    span = self.current_span()

return DDLogRecord(... what you have...)
```

could be a way to get rid of the `Optional` part.

What do you think?